endpoints/openrtb2/video_auction_test.go

```diff
@@ -106,10 +106,12 @@ func TestCreateBidExtension(t *testing.T) {
 		Increment: 0.1,
 	})
 
+	translateCat := true
```

Comment:
Consider renaming to translateCategories to match the naming of video_auction.go.

Reply:
Updated to `translateCategories`